src/WebPageTranscoder.js

```diff
@@ -21,10 +21,12 @@ import {ErrorReporter} from './util/ErrorReporter.js';
 export const scriptSelector = 'script[type="module"],script[type="text/traceur"]';
 
 export class WebPageTranscoder {
-  constructor(url) {
+  constructor(url = document.location.href,
+      traceurOptions = $traceurRuntime.options) {
```

Comment:
Should this be?

```js
constructor(url = document.location.href,
            options = new Options()) {
```

I would like us to get away from using `$traceurRuntime.options` if possible.